### DISCUSSION TITLE
Installing on Windows

### DISCUSSION THREAD

**Discussion by Promootheus** [metavoiceio/metavoice-src#77] [Q&A]
Does anyone have a wokrking process for getting this working in Windows or this Linux only?

**Comment by danablend** [comment]
> Does anyone have a wokrking process for getting this working in Windows or this Linux only?

Not sure if this helps, but I use Windows and use WSL2 to run a Linux subsystem - this works tremendously well for me :-)

**Reply by Promootheus** [reply]
Thanks, that's good to know. I wasn't sure if I'd run into any GPU related issues with WSL so ended up dual booting, but it's really helpful to know this works well with WSL.

**Reply by sidroopdaska** [reply]
@Promootheus did Danablends comment work for you? Can I mark his comment as answer and close this discussion?

**Comment by bigbrotherr** [comment]
I installed using Docker on windows with no issues
